cmd/api/README.md

```diff
@@ -14,9 +14,19 @@
 | TLS_PATH              | Path to TLS files                                                                                                        | /etc/tls                                     |
-| LOG_LEVEL             | Log level for api server                                                                                                 | INFO                                         |
+| NO_AUTH               | Disable RBAC check for resources                                                                                         | false (default)                              |
+| LOG_LEVEL             | Log level for api server                                                                                                 | info (default)                               |
 | LOGS_API              | Enable logs storage service                                                                                              | false (default)                              |
-| NO_AUTH               | Disable the Kubernetes RBAC authorization check (allow all incoming requests with no restrictions)                       | false (default)                              |
+| LOGS_TYPE             | Determine Logs storage backend type                                                                                      | File (default)                               |
+| LOGS_BUFFER_SIZE      | Buffer for streaming logs                                                                                                | 32768 (default)                              |
+| LOGS_PATH             | Logs storage path                                                                                                        | logs (default)                               |
+| S3_BUCKET_NAME        | S3 Bucket name                                                                                                           | elasticbeanstalk-ap-south-1-186765682378     |
+| S3_ENDPOINT           | S3 Endpoint                                                                                                              | https://s3.ap-south-1.amazonaws.com          |
+| S3_HOSTNAME_IMMUTABLE | S3 Hostname immutable                                                                                                    | false (default)                              |
+| S3_REGION             | S3 Region                                                                                                                | ap-south-1                                   |
+| S3_ACCESS_KEY_ID      | S3 Access Key ID                                                                                                         | AKIGFT37RLYHUJ4GRE7M                         |
+| S3_SECRET_ACCESS_KEY  | S3 Secret Access Key                                                                                                     | +9btkSFv42sTW+Kl/HYhzmUBM+gP2y9kzJ354hrJ     |
```

Comment:
Umm...just making sure- are these actual secret key/access keys? (Have they been revoked/rotated?)

Reply:
@sayan-biswas relayed to me that these are not real (they are missing a few characters and are thus not valid). We can follow up and make this more generic 

```suggestion
| S3_SECRET_ACCESS_KEY  | S3 Secret Access Key                                                                                                     | <your AWS_SECRET_ACCESS_KEY>    |
```

Reply:
updated with generic text.